docs/quick-start.md

```diff
@@ -8,6 +8,8 @@ Firstly, you'll need a Kubernetes cluster and `kubectl` set-up
 
 To get started quickly, you can use the quick start manifest which will install Argo Workflow as well as some commonly used components:
 
+!!! note
+    For ease of use, this will configure PostgreSQL and MinIO with basic and well-known passwords. We recommend that you change these to be more secure at your earliest opportunity.       
```

Comment:
I think it would be more correct to say

These manifests are intended to help you get started quickly. They are not suitable in production, on test environments, or any environment containing any real data. They contain hard-coded passwords that are publicly available.